modules/features/filters/build.gradle.kts

```diff
@@ -37,9 +37,16 @@ dependencies {
     api(projects.modules.services.views)
     api(projects.modules.services.compose)
 
+    implementation(platform(libs.compose.bom))
+
     implementation(libs.androidx.constraintlayout)
     implementation(libs.androidx.core.ktx)
     implementation(libs.androidx.preference.ktx)
+    implementation(libs.compose.material)
+//    implementation(libs.compose.material3.window.size)
```

Comment:
Do we need to keep this commented out library included?

Reply:
No, it came from a stash and I forgot to remove it. I'll remove it.